src/test/java/codeu/model/store/persistence/PersistentStorageAgentTest.java

```diff
@@ -44,6 +46,12 @@ public void testLoadMessages() throws PersistentDataStoreException {
     Mockito.verify(mockPersistentDataStore).loadMessages();
   }
 
+  @Test
+  public void testLoadHashtags() throws PersistentDataStoreException {
```

Comment:
You should add a description of this function (loadHashtags) above it , just like the other ones.